tests/modules.py

```diff
@@ -314,6 +314,19 @@ def test_npm_app(network, args):
         body = r.body.json()
         assert body["msg"] == "Hello!", r.body
 
+        r = c.post("/app/log?id=42", {"msg": "Saluton!"})
```

Comment:
Probably should change this to `/logs` and `/logs/{id}` now that JS supports templates (maybe in another PR, just mentioning it since you added `/log/all`).